### ISSUE TITLE
Can't navigate tag field dropdowns using keyboard

### ISSUE THREAD

**Issue by proimage** [craftcms/cms#2921]
### Description
When typing a tag, be it a new one to add or an existing one to select, the dropdown that appears with possible matches is not accessible to keyboard-only users.

### Steps to reproduce

1. Create a Tag field
2. Add said field to a section.
3. Create new entry in said section
4. Type something in the tag field and try to select a result from the dropdown using the keyboard only.

### Additional info

- Craft version: 3.0.8
